src/encoded/types/biosample.py

```diff
@@ -778,7 +773,7 @@ def generate_modification_summary(method, modification):
 
         if modification.get('target'):
             modification_summary += ' targeting ' + modification.get('target')
- 
+
```

Comment:
We shouldn't make changes not related to the feature.

Reply:
done